src/utils/node/pm2.ts

```diff
@@ -0,0 +1,129 @@
+import * as path from 'path';
+import { connect, describe, disconnect, list, ProcessDescription, restart, start, stop } from 'pm2';
+
+const connectPM2 = async (): Promise<void> =>
+    new Promise<void>((resolve, reject) => {
+        connect(err => {
+            if (err) {
+                reject(err);
+
+                return;
+            }
+            resolve();
+        })
+    })
+
+const startPM2 = async (installPath: string, name: string): Promise<void> =>
+    new Promise<void>((resolve, reject) => {
+        start({
+            name,
+            script: 'app.js',
+            args: '-c config.json',
+            cwd: installPath,
+            pid: path.join(installPath, '/pids/lisk.app.pid'),
+            output: path.join(installPath, '/logs/lisk.app.log'),
+            error: path.join(installPath, '/logs/lisk.app.err'),
+            log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
+            watch: false,
+            kill_timeout: 10000,
+            max_memory_restart: '1024M',
+            min_uptime: 20000,
+            max_restarts: 10,
+        }, err => {
+            if (err) {
+                reject(err);
```

Comment:
`return reject(err);` this should work, and same for `return resolve();`

Reply:
I think this is one of tslint rule that return type must match and explicit.

Reply:
ahhh ok 👍